### PR TITLE
Made modal images opening

### PR DESCRIPTION
**Pull request by Saiwi**
- Made opening images in modal window on post page in theme "Splynx"
- Added nav-header in theme "Splynx"

### CODE REVIEW

**Review by andriisimon — APPROVED** [review]

File: client/components/admin.vue [hunk 1]
```diff
@@ -249,7 +249,7 @@ export default {
         this.$store.commit(`loading${isLoading ? 'Start' : 'Stop'}`, 'admin-stats-refresh')
       }
     }
-  }
+  },
```
Comment (mishchanchukspl) [inline client/components/admin.vue]: It is better to delete this change

**Review by mishchanchukspl — COMMENTED** [review]
esc does not close the modal window. I think it should

File: client/components/admin.vue (hunk 1 above)
Comment (Saiwi) [reply client/components/admin.vue]: Fixed

**Comment by Saiwi** [comment]
> esc does not close the modal window. I think it should

Fixed

**Comment by mishchanchukspl** [comment]
> Fixed

Not fixed if modal not focused, modal don't close

**Comment by Saiwi** [comment]
> > Fixed
> 
> Not fixed if modal not focused, modal don't close

Fixed

**Pull request merged**